crates/miden-tx/src/executor/notes_checker.rs

```diff
@@ -144,20 +144,21 @@ where
         &self,
         target_account_id: AccountId,
         block_ref: BlockNumber,
-        note: InputNote,
+        note: Note,
```

Comment:
As per https://github.com/0xMiden/miden-base/issues/1936#issuecomment-3408385227, it's probably best to keep this as `InputNote`